src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.util.StringJoiner;
+
+import java.time.LocalDateTime;
```

Comment:
Good use of the java Time classes to format your time neatly, without confusing any reader. I personally implemented a method just to convert the string into a time format, but it just adds unnecessary code for any reviewer.